packages/components/pie-web-components-core/README.md

```diff
@@ -0,0 +1,7 @@
+<p align="center">
+  <img align="center" src="../../../readme_image.png" height="200" alt="">
+</p>
+
+# pie-web-components-core
```

Comment:
To keep the naming with our docs site (and because I'm lazy) – would this be potentially better named `pie-webc-core`

Reply:
sounds good!